admin_guide/tcp_ingress_external_ports.adoc

```diff
@@ -0,0 +1,99 @@
+[[admin-guide-unique-external-ips-ingress-traffic]]
+= Assigning Unique External IPs for Ingress Traffic
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Cluster administrators can assign a unique IP address to a service that, if
+routed correctly, enables external traffic to reach the ports of that service.
```

Comment:
the ports of that service -> that service's endpoints via any TCP and UDP port the service exposes. ?